internal/cmd/commands/credentialstorescmd/funcs.go

```diff
@@ -0,0 +1,203 @@
+			)
+		} else {
+			output = append(output,
+				fmt.Sprintf("  ID:                    %s", "(not available)"),
+			)
+		}
+		if c.FlagRecursive && m.ScopeId != "" {
+			output = append(output,
+				fmt.Sprintf("    Scope ID:            %s", m.ScopeId),
+			)
+		}
+		if m.Version > 0 {
+			output = append(output,
+				fmt.Sprintf("    Version:             %d", m.Version),
+			)
+		}
+		if m.Type != "" {
+			output = append(output,
+				fmt.Sprintf("    Type:                %s", m.Type),
+			)
+		}
+		if m.Name != "" {
+			output = append(output,
+				fmt.Sprintf("    Name:                %s", m.Name),
+			)
+		}
+		if m.Description != "" {
+			output = append(output,
+				fmt.Sprintf("    Description:         %s", m.Description),
+			)
+		}
+		if len(m.AuthorizedActions) > 0 {
+			output = append(output,
+				"    Authorized Actions:",
+				base.WrapSlice(6, m.AuthorizedActions),
+			)
+		}
+	}
+
+	return base.WrapForHelpText(output)
+}
+
+func printItemTable(result api.GenericResult) string {
+	item := result.GetItem().(*credentialstores.CredentialStore)
+	nonAttributeMap := map[string]interface{}{}
+	if item.Id != "" {
+		nonAttributeMap["ID"] = item.Id
+	}
+	if item.Version != 0 {
+		nonAttributeMap["Version"] = item.Version
+	}
+	if !item.CreatedTime.IsZero() {
+		nonAttributeMap["Created Time"] = item.CreatedTime.Local().Format(time.RFC1123)
+	}
+	if !item.UpdatedTime.IsZero() {
+		nonAttributeMap["Updated Time"] = item.UpdatedTime.Local().Format(time.RFC1123)
+	}
+	if item.Name != "" {
+		nonAttributeMap["Name"] = item.Name
+	}
+	if item.Description != "" {
+		nonAttributeMap["Description"] = item.Description
+	}
+	if item.Type != "" {
+		nonAttributeMap["Type"] = item.Type
+	}
+
+	maxLength := base.MaxAttributesLength(nonAttributeMap, item.Attributes, keySubstMap)
+
+	ret := []string{
+		"",
+		"Credential Store information:",
+		base.WrapMap(2, maxLength+2, nonAttributeMap),
+	}
+
+	if item.Scope != nil {
+		ret = append(ret,
+			"",
+			"  Scope:",
+			base.ScopeInfoForOutput(item.Scope, maxLength),
+		)
+	}
+
+	if len(item.AuthorizedActions) > 0 {
+		ret = append(ret,
+			"",
+			"  Authorized Actions:",
+			base.WrapSlice(4, item.AuthorizedActions),
+		)
+	}
+
+	if len(item.AuthorizedCollectionActions) > 0 {
+		keys := make([]string, 0, len(item.AuthorizedCollectionActions))
+		for k := range item.AuthorizedCollectionActions {
+			keys = append(keys, k)
+		}
+		sort.Strings(keys)
+		ret = append(ret, "",
+			"  Authorized Actions on Credential Store's Collections:",
+		)
+		for _, key := range keys {
+			ret = append(ret,
+				fmt.Sprintf("    %s:", key),
+				base.WrapSlice(6, item.AuthorizedCollectionActions[key]),
+			)
+		}
+	}
+
+	if len(item.Attributes) > 0 {
+		ret = append(ret,
+			"",
+			"  Attributes:",
+			base.WrapMap(4, maxLength, item.Attributes),
+		)
+	}
+
+	return base.WrapForHelpText(ret)
+}
+
+var keySubstMap = map[string]string{
+	"address":                     "Address",
```

Comment:
The substitutions in these maps should be "human friendly" so things like "TLS Server Name", "CA Cert", etc. @talanknight